lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -217,6 +208,53 @@ def el8?
       def el_major_version
         return os_facts['os']['release']['major'] if el?
       end
+
+      def ruby_prefix(scl = true)
+        if el7? && scl
+          'tfm-rubygem-'
+        elsif el7? || el8?
+          'rubygem-'
+        elsif debian?
+          'ruby-'
+        end
+      end
+
+      def foreman_plugin_name(plugin)
+        if debian?
+          plugin.tr!('_', '-')
+        end
+        ruby_prefix + plugin
+      end
+
+      def proxy_plugin_name(plugin)
+        if debian?
+          plugin.tr!('_', '-')
+          proxy_plugin_prefix = 'smart-proxy-'
+        else
+          proxy_plugin_prefix = 'smart_proxy_'
+        end
+        scl = check_min_version('foreman', '2.0')
+        ruby_prefix(scl) + proxy_plugin_prefix + plugin
+      end
+
+      def hammer_package
+        if el7?
+          'tfm-rubygem-hammer_cli'
+        elsif el8?
+          'rubygem-hammer_cli'
+        elsif debian?
+          'rubygem-hammer-cli'
+        end
```

Comment:
```suggestion
        if debian?
          hammer_prefix = 'hammer-cli'
        else
          hammer_prefix = 'hammer_cli'
        end
        ruby_prefix + hammer_prefix
```

(the original code used `rubygem` for Debian, which is wrong, and then I was like "wait, we have a helper for that!")